.gitlab-ci.yml

```diff
@@ -270,6 +280,41 @@ trigger:saas:sync-staging-component:
   rules:
     - when: never
 
+trigger:staging:cleanup-tenants:
+  image: alpine:3.13
+  stage: .post
+  rules:
+    - if: $RUN_TESTS_STAGING == "true"
+      when: always
+
+  variables:
+    K8S: "staging"
+    AWS_ACCESS_KEY_ID: $CI_JOBS_AWS_ACCESS_KEY_ID
+    AWS_SECRET_ACCESS_KEY: $CI_JOBS_AWS_SECRET_ACCESS_KEY
+    AWS_DEFAULT_REGION: $CI_JOBS_AWS_REGION
+    AWS_EKS_CLUSTER_NAME: $CI_JOBS_AWS_EKS_CLUSTER_NAME
```

Comment:
Related to @alfrunes comment, I think we can safely export these in `before_script` and remove them from here, to be more explicit and avoid possible overrides of `AWS_DEFAULT_REGION` or `AWS_EKS_CLUSTER_NAME` in the future.